emcc.py

```diff
@@ -1512,7 +1521,10 @@ def repl(m):
         if DEBUG:
           # Copy into temp dir as well, so can be run there too
           shared.safe_copy(memfile, os.path.join(shared.get_emscripten_temp_dir(), os.path.basename(memfile)))
-        return 'memoryInitializer = "%s";' % os.path.basename(memfile)
+        if shared.Settings.CYBERDWARF:
+          return 'memoryInitializer = "%s"; emDebugCDFile = "%s";' % (os.path.basename(memfile), os.path.basename(memfile)[:-3] + "cd")
```

Comment:
To pass it into the emitted JS? Define a variable in settings.js, set shared.Settings.THAT_VAR in emcc.py, and then you can use that variable either in emscripten.py (on settings) or in the JS files, for example, you can do `{{{ THAT_VAR }}}`, or `#if THAT_VAR` etc, in preamble.js.